.canon.yaml

```diff
@@ -1,6 +1,6 @@
 rplidar:
-    image_amd64: ghcr.io/viamrobotics/canon:amd64
-    image_arm64: ghcr.io/viamrobotics/canon:arm64
+    image_amd64: ghcr.io/viam-modules/canon:amd64
+    image_arm64: ghcr.io/viam-modules/canon:arm64
```

Comment:
FYI, image addresses haven't moved, so all the changes like this throughout that are pointing to canon shouldn't be changed.

Also, the old "canon" image is way out of date now. If it still works, that's fine. But the maintained one (from RDK) is `ghcr.io/viamrobotics/rdk-devenv`

Reply:
great catch, thanks james!

Reply:
cc @hexbabe I was just looking at the real sense repo and I noticed that you also changed the canon addresses to `viam-modules`, so you might need to change them. (apologies if the real sense doesn't use the same canon images and this is irrelevant but thought I'd double check)